dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -1315,7 +1315,6 @@ protected boolean checkAndImport(User loginUser, long projectCode, Map<String, O
             return false;
         }
         processDefinition.setName(importProcessDefinitionName);
-        processDefinition.setId(0);
```

Comment:
When import workflow, it should create a new id, so it's better to setId(null), but not use the origin id.